walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -4,68 +4,105 @@
     <meta charset="UTF-8">
     <title>All About Jalyn 🐈</title>
     <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
   </head>
   <body>
-<div class="navigation_bar">
-    
+    	<div id="navigation-bar">
+        	<span class="nav nav-btn"> <a class="nav" href= "index.html" style="background-color: white">Home</a> </span>
+        	<span class="nav nav-btn"> <a class="nav" href="Resume_CS.pdf" style="background-color: white">Resume</a> </span>
+        	<span class="nav nav-btn"> <a class="nav" href="/socials.html" style="background-color: white">Socials</a> </span>
+    	</div>
 
-	<h3><a class="nav" href="/random.html">Link One</a></h3>
+        <div class="full-page">
+            <div id="first-page">
+                <h1>The One Where Jalyn Makes A Portfolio...</h1>
+                <h2> We're about to get real close</h2>
+                <img src="/images/hello.jpeg" id="top-img">
+            </div>
+        </div>
+
+
+        <div id="second-page" class= "full-page">
+            <ul class="caption-style-1">
+                <li> <img src="/images/new-zealand.jpg" alt="">
+                <div class="caption">
+                    <div class="blur"></div>
+                    <div class="caption-text"
+                            <h1>Desciption: Image of the map of New Zealand</h1>
+                            <p>The place I want to visit more than anywhere in the world, would
+                                be the country of New Zealand. The beautiful nature, and the most
+                                tropical climate, makes this the best travel destination!
+                            </p>
+                    </div>
+                </div>
+                </li>
+
+                <li> <img src="/images/profilepic.jpg" alt="">
```

Comment:
add alt text in case images don't render.

Reply:
Done!